.github/workflows/scripts/e2e.generic.default.verify.sh

```diff
@@ -5,8 +5,9 @@ source "./.github/workflows/scripts/e2e-utils.sh"
 go env -w GOFLAGS=-mod=mod
 
 # Install from HEAD
-# go install github.com/slsa-framework/slsa-verifier@latest
-go install github.com/ianlewis/slsa-verifier@ianlewis-test
+go install github.com/slsa-framework/slsa-verifier@latest
+
+THIS_FILE=$(gh api -H "Accept: application/vnd.github.v3+json" "/repos/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" | jq -r '.path' | cut -d '/' -f3)
```

Comment:
I think we could have a function instead of copying this line in every file. (My fault I did not do it the first time). Not need for this PR, just fyi in case you find the time to do it before me :-)

Reply:
Yeah, We talked about grouping up the common checks into it's own file. I think this should be done there. I think #25 is the issue.

Reply:
I also created https://github.com/slsa-framework/slsa-github-generator/issues/152 I think for the same thing.